src/firefly/js/visualize/ui/ColorBandPanel.jsx

```diff
@@ -370,16 +344,83 @@ function renderGamma(fields) {
     );
 }
 
-function renderAsinH(fields) {
-    var {zscale}= fields;
-    var range= (zscale.value==='zscale') ? renderZscale() : getUpperAndLowerFields();
+const asinhSliderMarks = {
+ 0: '0', 5: '5', 10: '10', 15: '15', 20: '20', 25: 'InF'
```

Comment:
My opinion is that the label 'Inf' should be '25'. The text above the slider says that Q = Inf is log stretch and I think that conveys to a user that large values are a log stretch. I think 25 is not close enough to a very large value for us to call this 'Inf'.

I could be talked out of this though so maybe it is better to leave this as-is and get opinions from more people.